src/bots/meets/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "teams",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.ts",
+  "scripts": {
+    "dev": "tsx src/index.ts",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "preinstall": "pnpm dlx puppeteer browsers install chrome"
```

Comment:
If we're using playwrite we can remove the puppeteer preinstall here

Reply:
Switched from puppeteer installation to playwright